src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java

```diff
@@ -0,0 +1,21 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OuterTypeNumber"/>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;// violation, Outer types defined is '3' (max allowed is '1')
+
+public class Example1 {
```

Comment:
Please make it:
```
package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
// xdoc section -- start
// violation above, Outer types defined is '3' (max allowed is '1')

public class Example1 ......
```

Reply:
Same exclude of package for example2

Reply:
Package is too long. Violation is not visible. It took me while to understand why I don't see violation

Reply:
@romani ,done please review the items